main.tf

```diff
@@ -166,6 +166,9 @@ resource "aws_s3_bucket_versioning" "cloudtrail_bucket_versioning" {
   }
 }
 
+data "aws_organizations_organization" "organization" {
```

Comment:
Can we validate that this "data source" add some new permission to our prefilght checklists?